modules/nf-core/gnu/sort/environment.yml

```diff
@@ -1,7 +1,9 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
 name: gnu_sort
 channels:
   - conda-forge
   - bioconda
   - defaults
 dependencies:
-  - bioconda::coreutils=8.25
+  - conda-forge::coreutils=9.3
```

Comment:
Why the conda forge one? It's better from a Conda perspective, but it takes things out of synch with Docker. I think it would be best to bump it but keep to the Bioconda one.

Reply:
The problem here is that the package is not in bioconda any more (does not exist in the repo). I don't know how the biocontainers received the update to 9.3, but the latest version in [bioconda](https://anaconda.org/bioconda/coreutils) is 8.25.

Reply:
I guess they did this for the same reason in [gnu_split](https://github.com/nf-core/modules/tree/master/modules/nf-core/gnu/split), but it was not discussed in #5428

Reply:
Ahh, as you were then :-)